setup.py

```diff
@@ -92,6 +92,7 @@ def load_deps(path):
     ],
     "plotly": ["plotly~=5.4"],
     "google-cloud-storage": ["gcsfs~=2021.8.1"],
+    "google-cloud-bigquery": ["google-cloud-bigquery~=3.0"],
```

Comment:
Since you added a new extras package, please add it to the package-test (in `automation/package_test/test.py`). Alternatively, maybe this should sit in the same package as `google-cloud-storage`? (which will also require a package-test change, to add another `import` there).